packages/automl/frontend/src/app/components/run-results/StepDetailsPanel.scss

```
@@ -0,0 +1,39 @@
+.automl-step-details__empty-state-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  max-width: 320px;
+  padding: var(--pf-t--global--spacer--lg);
+}
+
+.automl-step-details__empty-state-spinner {
+  --pf-v6-c-spinner--diameter: 4rem;
+  margin-bottom: var(--pf-t--global--spacer--lg);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)